DataScience/ExperimentationAzure.py

```diff
@@ -209,6 +210,4 @@ def check_system():
         Logger.info('Total Job time in seconds: {}'.format((end_time - start_time).seconds))
         sys.stdout.flush()
         sys.stderr.flush()
-        azure_util.upload_to_blob(ld_args.app_id, os.path.join(main_args.output_folder, 'stdout.txt'), os.path.join(task_dir, 'stdout.txt'))
```

Comment:
I supposed this was for the debug purpose - just to double check, what would be the new way to check the log?

Reply:
The stdout.txt and stderr.txt files stay in the task directory by default, so you can access them by going to our azure batch account -> jobs -> <relevant job> -> tasks and then looking at the files for that task. The reason we could not see them before is that we had set up the job release script for our jobs to delete all files in the task directory, but I'm removing that behavior. The caveat here is that the files will only be available for 7 days by default, which we can discuss increasing. All in all, I prefer this method as eventually we will not be able to access customer storage w/the lockbox changes